MVC-App/Services/RelationService.cs

```diff
@@ -200,36 +193,40 @@ public async Task<RelationListVM> GetAsync(Guid? categoryId)
 
         public async Task Create(CreateEditRelationVM relationVM)
         {
-            var relation = new Relation
-            {
-                Id = Guid.NewGuid(),
-                Name = relationVM.Relation.Name,
-                FullName = relationVM.Relation.FullName,
-                TelephoneNumber = relationVM.Relation.TelephoneNumber,
-                EMailAddress = relationVM.Relation.Email,
-                CreatedAt = DateTime.Now,
-                CreatedBy = "admin",
-                IsDisabled = false,
-                IsTemporary = false,
-                IsMe = false,
-                PaymentViaAutomaticDebit = false,
-                InvoiceDateGenerationOptions = 0,
-                InvoiceGroupByOptions = 0
-            };
+            var config = new MapperConfiguration(cfg => cfg.CreateMap<RelationVM, Relation>()
+            .AfterMap((src, dest) => {
+                dest.Id = Guid.NewGuid();
+                dest.EMailAddress = src.Email;
```

Comment:
Remove
dest.EMailAddress = src.Email;
And test if it works.

Reply:
it was added cause automapper fails to map this field